sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1236,6 +1260,12 @@ export class ContainerClient extends StorageClient {
     }>;
 }
 
+// @public
+export interface ContainerCpkScopeInfo {
+    defaultEncryptionScope?: string;
+    denyEncryptionScopeOverride?: boolean;
```

Comment:
Java/.NET has `PreventEncryptionScopeOverride`.  We should pick one that's consistent

Reply:
Let's align on `Prevent`.

Reply:
Will do.